src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -7,6 +7,7 @@
 
     public class When_subscribing_to_command_bestpractices_disabled_on_endpoint : NServiceBusAcceptanceTest
     {
+        [Ignore("not supported via code first API")]
```

Comment:
I was mostly wondering where that logic came from, because disabling best practices doesn't actually disable most of the potential warnings as it looks.